R/bundle.R

```diff
@@ -201,19 +201,22 @@ listBundleFiles <- function(appDir) {
 
 bundleFiles <- function(appDir) {
   files <- listBundleFiles(appDir)
-  if (files$totalSize > getOption("rsconnect.max.bundle.size")) {
+  enforceBundleLimits(appDir, files$totalSize, length(files$contents))
+  files$contents
+}
+
+enforceBundleLimits <- function(appDir, totalSize, totalFiles) {
```

Comment:
When writing a helper function that emits errors, I'd recommend setting `call. = FALSE` on the `stop()` calls.

Reply:
Thanks. Done. Updated the description to reflect the updated error formatting.